Makefile

```diff
@@ -50,8 +50,13 @@ ifndef message
 endif
 	poetry run flask db revision -m "$(message)" --autogenerate
 
-TESTS ?= ./tests/
+TESTS ?= ./tests
 .PHONY: test
 test: ## Run the test suite
 	docker compose run --rm app \
 		poetry run pytest --cov hushline --cov-report term --cov-report html -vv $(PYTEST_ADDOPTS) $(TESTS)
+
+.PHONY: test/ui
+test/ui: # Only run the ui tests
```

Comment:
I think we way we'll want to split this is either using some kind of `pytest.mark` or possibly a wholly separate directory like `ui-test` since we'd need a separate set of fixtures for a live app, though there might be some clean way to make them overlap.